proc.c

```diff
@@ -1840,6 +1840,28 @@ method_eq(VALUE method, VALUE other)
     return Qtrue;
 }
 
+#if 0 /* for RDoc */
+/*
+ * call-seq:
+ *   meth.eql?(other_meth)  -> true or false
+ *   meth == other_meth  -> true or false
+ *
+ * Two unbound method objects are equal if they refer to the same
+ * method definition.
+ *
+ *    Array.instance_method(:each_slice) == Enumerable.instance_method(:each_slice)
+ *    #=> true
+ *
+ *    Array.instance_method(:sum) == Enumerable.instance_method(:sum)
+ *    #=> false, Array redefines the method for efficiency
+ */
+
+static VALUE
+unbound_method_eq(VALUE method, VALUE other)
+{
+}
+#endif
```

Comment:
RDoc handles method definitions by macros.
```suggestion
#define unbound_method_eq method_eq
```

Reply:
Ugh, right, I forgot you can do this :bow: